saleor/payment/gateways/stripe/consts.py

```diff
@@ -1,10 +1,18 @@
 PLUGIN_ID = "saleor.payments.stripe"
 PLUGIN_NAME = "Stripe"
 WEBHOOK_PATH = "webhooks/"
+
+WEBHOOK_SUCCESS_EVENT = "payment_intent.succeeded"
+WEBHOOK_PROCESSING_EVENT = "payment_intent.processing"
+WEBHOOK_FAILED_EVENT = "payment_intent.payment_failed"
+WEBHOOK_AUTHORIZED_EVENT = "payment_intent.amount_capturable_updated"
```

Comment:
Does `amount_capturable_updated` equals `authorized` in Stripe? Is that the only case it's being triggered?

Reply:
![image](https://user-images.githubusercontent.com/23282005/119620841-bd998180-be05-11eb-9b01-d60b366656a0.png)
https://stripe.com/docs/payments/payment-intents/verifying-status#handling-webhook-events